src/Helper/DrupalApiHelper.php

```diff
@@ -63,6 +67,21 @@ public function getCreateVocabularies()
 
         return $createVocabularies;
     }
+
+    /**
+     * @return \Drupal\Console\Utils\Create\Nodes
```

Comment:
method must return 
`\Drupal\Console\Utils\Create\Users`
instead of
`\Drupal\Console\Utils\Create\Nodes`